docs/overview/concepts/querying.md

```diff
@@ -105,3 +105,7 @@ Quickwit does caching in many places to deliver a highly performing query engine
 ### Scoring
 
 Quickwit supports sorting docs by their BM25 scores. In order to query by score, [fieldnorms](../../configuration/index-config.md#text-type) must be enabled for the field. By default, BM25 scoring is disabled to improve query latencies but it can be opt-in by setting the `sort_by` option to `_score` in queries.
+
+### Document ID
+
+Each document in Quickwit is assigned a unique document ID which is a combination of the split ID and the Tantivy DocId within the split. You cannot assign a custom ID. This ID is used for every search query as sort order (after the explicitly specified sort values) to make the results deterministic.
```

Comment:
I think it is worth defining the doc id here, especially for seasoned ES users who might be looking into how this works in Quickwit. I'm open to suggestions on the level of details we should go into, as this is _mostly_ an internal concept.

Reply:
i guess you could add that this id is not stable across merges, current wording might suggest it is (i don't think it's necessarily obvious that split id changes after merges).
I don't know if saying it's a combination of split id and tantivy docid is actually useful

Reply:
> I don't know if saying it's a combination of split id and tantivy docid is actually useful

yeah it's kind of an implementation detail, but I think it helps figuring out how they behave